<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tttAddFrameDialog</class>
 <widget class="QDialog" name="tttAddFrameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="addToLayerLabel">
     <property name="text">
      <string>Add To Layer:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="layerComboBox"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableWidget" name="selectedFilesWidget">
     <property name="columnCount">
      <number>8</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="cautionLabel">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string>CAUTION: Existing frames in the project will be overwritten!</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="addFilesButton">
     <property name="text">
      <string>Add Files...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="selectedFilesLabel">
     <property name="text">
      <string>Files to Add:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 255, 0);</string>
     </property>
     <property name="text">
      <string>Remember to adjust timestamp information</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tttAddFrameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tttAddFrameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
